.github/workflows/ci.yml

```diff
@@ -4,18 +4,23 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: macos-latest
+    runs-on: self-hosted
```

Comment:
wow. great that we can use github actions with a self-hosted environment. didn't expect that!

Reply:
@dfed if you're moving one of your repos I highly recommend looking at https://youtu.be/9RQxeRXHQZM

Happy to help too btw.